src/engraving/compat/read114.cpp

```diff
@@ -2828,7 +2829,7 @@ Score::FileError Read114::read114(MasterScore* masterScore, XmlReader& e)
                 if (e.name() == "tempo") {
                     int tick   = e.attribute("tick").toInt();
                     double tmp = e.readElementText().toDouble();
-                    tick       = (tick * MScore::division + masterScore->_fileDivision / 2) / masterScore->_fileDivision;
+                    tick       = ctx.fileDivision(tick);
```

Comment:
isn't there some calculations lost here?

Reply:
no, it's ok, this method will be called https://github.com/musescore/MuseScore/blob/master/src/engraving/libmscore/score.h#L886